airbyte-integrations/connectors/source-jdbc/src/testFixtures/java/io/airbyte/integrations/source/jdbc/test/JdbcSourceStandardTest.java

```diff
@@ -84,18 +85,35 @@
 // 4. Then implement the abstract methods documented below.
 public abstract class JdbcSourceStandardTest {
 
-  private static final String SCHEMA_NAME = "jdbc_integration_test";
-  private static final String SCHEMA_NAME2 = "jdbc_integration_test2";
-  private static final Set<String> TEST_SCHEMAS = ImmutableSet.of(SCHEMA_NAME, SCHEMA_NAME2);
-
-  private static final String TABLE_NAME = "id_and_name";
-  private static final String TABLE_NAME_WITHOUT_PK = "id_and_name_without_pk";
-  private static final String TABLE_NAME_COMPOSITE_PK = "full_name_composite_pk";
-
-  private JsonNode config;
-  private JdbcDatabase database;
-  private AbstractJdbcSource source;
-  private static String streamName;
+  public static String SCHEMA_NAME = "jdbc_integration_test1";
+  public static String SCHEMA_NAME2 = "jdbc_integration_test2";
+  public static String SCHEMA_NAME3 = "jdbc_integration_test3";
```

Comment:
why do we need so many different schema names? we only had 2 before. i'm a bit confused how the extra ones are being used.

Reply:
I created a function called `cleanUpTables` and remove all extra schemas.